internal/config/application_test.go

```diff
@@ -96,25 +86,31 @@ func TestApplicationConfig(t *testing.T) {
 			assertions: func(t *testing.T, app *Application) {
 				assert.Equal(t, "test-home-XDG-config", app.File)
 			},
-			Cleanup: func(t *testing.T) {},
+			cleanup: func() {
+				require.NoError(t, os.Unsetenv("XDG_CONFIG_DIRS"))
```

Comment:
This might actually need to track the original value to accurately reverse the behavior

Reply:
good call